tools/isolib/isolib.xml

```diff
@@ -58,7 +59,16 @@
                 </param>
             </when>
         </conditional>
-        <param name="threshold" type="float" min="0" max="100" value="1" label="Threshold" help="Probability threshold to use as cutoff for isotopic pattern distribution - this can be used to remove low abundant peaks and improve computation performance." />
+        <param name="threshold" type="float" min="0.0001" max="1.0" value="0.1"
+       label="Threshold"
+       help="Relative intensity threshold (0.0001 to 1.0) for isotopic peaks. Peaks below this fraction of the base peak will be excluded to reduce noise and improve performance." />
+       <param name="rel_to" type="select" value="0" label="Probability Reference" help="Defines how isotope peak probabilities are normalized for pruning — e.g., relative to the most intense, monoisotopic, or as absolute values.">
+            <option value="0">Most intense peak</option>
+            <option value="1">Most abundant isotopes per element</option>
+            <option value="2">Absolute probability</option>
+            <option value="3">Most intense peak (threshold as percentage)</option>
+            <option value="4">Monoisotopic peak (threshold as percentage)</option>
```

Comment:
Did you test whether this works then with the values from 0 to 1? Or do they have to be converted to % values by multiplying with 100? We should add a test for this.